src/Semantics.js

```diff
@@ -196,6 +196,41 @@ Semantics.prototype.checkActionDicts = function() {
   }
 };
 
+Semantics.prototype.toRecipe = function() {
+  var sb = new common.StringBuffer();
+  sb.append('(function() {\n  var grammarFn = ');
+  sb.append(this.grammar.toRecipe());
+  sb.append('  var grammar = grammarFn.call(this);\n');
+  sb.append('  var semantics = grammar.semantics()');
+
+  ['Operation', 'Attribute'].forEach(function(type) {
+    var semantics = this[type.toLowerCase() + 's'];
```

Comment:
I was wondering whether there is a nice word for that - despite o-pe-ra-tions-and-a-ttri-butes. ;-)
I checked the documentation but it doesn't exist. :-( Maybe there will also be more in the future, `operationsAndAttributesAndXAndY`... Can we can up with a nicer, general name? (I also think that Meixian had a similar problem when writing code for the semantics editor...)